modules/modes/starters.py

```diff
@@ -321,52 +392,17 @@ def step(self):
                                 context.emulator.press_button("A")
                             else:
                                 self.update_state(ModeStarterStates.EXIT_MENUS)
+                                context.message = "Checking if shiny..."
                                 continue
 
                         case ModeStarterStates.EXIT_MENUS:
-                            if not config.cheats.starters:
```

Comment:
Looks like `config.cheats.starters` will no longer do a fast/cheaty shiny check for Kanto starters?